### DISCUSSION TITLE
PDU modeling

### DISCUSSION THREAD

**Discussion by ciroiriarte** [netbox-community/devicetype-library#1382] [Q&A]
Hello!,

I'm trying to model a PDU, and I'm wondering how should the port to breaker be mapped, if there are missing attributes in the model or if it's relevant at all and should just be ignored.

A- It has a single 3 phase input, that is supported.
B- It has 30 x C13/C19 combined outlets, not sure how to model that.
C- Each phase has 10 outlets, also supported attribute
D- And there are 6 breakers, 5 ports depend on each of them (2 circuits per phase). This doesn't seen to be supported.

Model in question:
https://www.geistglobal.com/power-distribution-unit/NI30682

**Comment by danner26** [comment]
Hey @ciroiriarte so let me break this out a bit and I will do my best to answer

In regards to B, this one is difficult. This is a neat PDU but not a common thing from the data centers ive frequented. That being said, I dont really know how to model this. Let me reach out to the other maintainers and I will get back to you

In regards to D, this is not current something that can be tracked. Are the breakers status accessible via the management console/cli? Can you control the breakers via that management interface? Also, is tracking breaker status something that you really need? Besides on off, what is there to track?

**Reply by ciroiriarte** [reply]
Regarding D, I still don't know. I trying to get SNMP access via de facilities administrator, not available yet.

I'll update as soon as I get information.

Also found that it has a management module that can be exchanged, I believe that can be covered with the modules facility.

**Reply by danner26** [reply]
It looks like this has come up once before, for a type called `HDOT Cx` which is the C13/C19 combo. Jeremy rightfully called it the "robot face" lol.. Still discussing how to handle this at the moment

Let me know how D goes

**Reply by ciroiriarte** [reply]
> It looks like this has come up once before, for a type called `HDOT Cx` which is the C13/C19 combo. Jeremy rightfully called it the "robot face" lol.. Still discussing how to handle this at the moment
> 
> Let me know how D goes

I believe "robot face" is the nickname for C19. Challenge would be the combo, I don't believe that has a standard NEMA naming, or industry wide defacto nickname...

**Comment by ciroiriarte** [comment]
Probably there's no magic answer. I'm thinking about modeling all of them as C13, and have users modify the port type on demand to C19 as needed.

**Reply by danner26** [reply]
That might be your best bet for now unless there is a standardized name for the interface. Unfortunately ive seen it called multiple different things between differernt manufacturers
